yt/utilities/tests/test_minimal_representation.py

```diff
@@ -8,13 +8,17 @@
 
 G30 = "IsolatedGalaxy/galaxy0030/galaxy0030"
 
+old_serialize = None
 
-def setup():
+
+def setup_module():
+    global old_serialize
```

Comment:
it'd be nice to have a pytest fixture for patching yt config options...

Reply:
I'm not sure that fixtures are flexible enough to do that, but we might be able to achieve this with a context manager wrapped in a decorator ?

Reply:
i was thinking that a fixture that used the monkeypatch fixture internally might work  -- I was just playing around with making a copy of a YTConfig instance, modifying a value and then using monkeypatch to set `yt.config.ytcfg` to point to the modified copy and it seemed to work. But it was a little circuitous and a more general context manager would much clearer. 

In any case, a discussion for a different issue/PR :)